website/source/docs/guides/creating-certificates.html.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: "docs"
+page_title: "Creating Certificates"
+sidebar_current: "docs-guides-creating-certificates"
+description: |-
+  Learn how to create certificates for Consul.
+---
+
+# Creating Certificates
+
+Correctly configuring TLS can be a complex process, especially given the wide
+range of deployment methodologies. This guide will provide you with a
+production ready TLS configuration.
+
+~> Note that while Consul's TLS configuration will be production ready, key
+   management and rotation is a complex subject not covered by this guide.
+   [Vault][vault] is the suggested solution for key generation and management.
+
+The first step to configuring TLS for Consul is generating certificates. In
+order to prevent unauthorized cluster access, Consul requires all certificates
+be signed by the same Certificate Authority (CA). This should be a _private_ CA
+and not a public one like [Let's Encrypt][letsencrypt] as any certificate
+signed by this CA will be allowed to communicate with the cluster.
+
+~> Consul certificates may be signed by intermediate CAs as long as the root CA
+   is the same. Append all intermediate CAs to the `cert_file`.
+
+
+## Reference Material
+
+- [Encryption](/docs/agent/encryption.html)
+
+## Estimated Time to Complete
+
+20 minutes
+
+## Prerequisites
+
+This guide assumes you have [cfssl][cfssl] installed (be sure to install
+cfssljson as well).
+
```

Comment:
Insert a section header, `## Steps` before the `### Step 1:` (see the [guide template](https://docs.google.com/document/d/17zk1a_azbAp0ph4_D_xh8E-LJ6tIFR_5C967ik1Rj9c/edit#)) .